jaxlie/hints/__init__.py

```diff
@@ -20,8 +20,18 @@ class RollPitchYaw(NamedTuple):
     yaw: Scalar
 
 
+try:
+    # This is only exposed in `jax>=0.2.21`.
+    from jax.random import KeyArray
+except ImportError:
+    KeyArray = Any  # type: ignore
```

Comment:
Thanks @oliverdutton!!

Do you see any issue with this approach instead of an import from `jax._src.random`? If possible I'd prefer to avoid imports from private modules.